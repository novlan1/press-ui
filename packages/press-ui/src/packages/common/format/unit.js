import { isDef, IN_BROWSER, isNumeric } from '../utils/validator';

export function addUnit(value) {
  if (!isDef(value)) {
    return undefined;
  }

  value = String(value);
  return isNumeric(value) ? `${value}px` : value;
} // cache

let rootFontSize;

function getRootFontSize() {
  if (!rootFontSize) {
    const doc = document.documentElement;
    const fontSize = doc.style.fontSize || window.getComputedStyle(doc).fontSize;
    rootFontSize = parseFloat(fontSize);
  }

  return rootFontSize;
}

function convertRem(value) {
  value = value.replace(/rem/g, '');
  return +value * getRootFontSize();
}

function convertVw(value) {
  value = value.replace(/vw/g, '');
  return +value * window.innerWidth / 100;
}

function convertVh(value) {
  value = value.replace(/vh/g, '');
  return +value * window.innerHeight / 100;
}

export function unitToPx(value) {
  if (typeof value === 'number') {
    return value;
  }

  if (IN_BROWSER) {
    if (value.indexOf('rem') !== -1) {
      return convertRem(value);
    }

    if (value.indexOf('vw') !== -1) {
      return convertVw(value);
    }

    if (value.indexOf('vh') !== -1) {
      return convertVh(value);
    }
  }

  return parseFloat(value);
}
