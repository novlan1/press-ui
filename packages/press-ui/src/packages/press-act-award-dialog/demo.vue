<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActAwardDialog
        sub-btn-text="订阅"
        :show.sync="show"
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :issubscribe="isSubscribe"
        @clickSub="isSubscribe = !isSubscribe"
        @clickButton="show = false"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActAwardDialog from 'press-ui/press-act-award-dialog/press-act-award-dialog.vue';
// import PressActAwardDialog from 'src/lib/act-award-dialog';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-award-dialog/demo-helper/style-list';
// import 'src/lib/lib-style/act-award-dialog.css';

export default {
  components: {
    PressActAwardDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActAwardDialog', STYLE_LIST),
  ],
  data() {
    return {
      show: true,
      isSubscribe: true,
    };
  },

  mounted() {
  },
  methods: {
  },
};
</script>
<style scoped lang="scss">
</style>
