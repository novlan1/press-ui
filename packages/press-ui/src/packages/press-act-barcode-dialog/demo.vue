<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActBarcodeDialog
        :show.sync="show"
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :url="url"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActBarcodeDialog from 'press-ui/press-act-barcode-dialog/press-act-barcode-dialog.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-barcode-dialog/demo-helper/style-list';


export default {
  components: {
    PressActBarcodeDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActBarcodeDialog', STYLE_LIST),
  ],
  data() {
    return {
      show: true,
      url: 'https://www.qq.com',
    };
  },

  mounted() {
  },
  methods: {
  },
};
</script>
<style scoped lang="scss">
</style>
