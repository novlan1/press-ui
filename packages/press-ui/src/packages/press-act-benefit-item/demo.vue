<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <div class="component-wrap">
      <PressActBenefitItem
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        @clickGiftDetail="clickGiftDetail"
        @clickGetTaskReward="clickGetTaskReward"
        @clickMerc="clickMerc"
      />
    </div>

    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      :show-basic-usage="false"
      @onCustom="onCustom"
    />

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import PressActBenefitItem from 'press-ui/press-act-benefit-item/press-act-benefit-item.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-benefit-item/demo-helper/style-list';
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';


export default {
  components: {
    PressActBenefitItem,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActBenefitItem', STYLE_LIST, 'pvp-vertical'),
  ],
  data() {
    return {
    };
  },

  mounted() {
  },
  methods: {
    clickGiftDetail() {
      this.onGTip('[clickGiftDetail]');
    },
    clickGetTaskReward() {
      this.onGTip('[clickGetTaskReward]');
    },
    clickMerc() {
      this.onGTip('[clickMerc]');
    },
  },
};
</script>
<style scoped lang="scss">
.component-wrap {
  margin: 20px 0 50px;
}
</style>
