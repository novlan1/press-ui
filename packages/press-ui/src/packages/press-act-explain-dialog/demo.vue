<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActExplainDialog
        :show.sync="show"
        :show-button="true"
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :props-data="propsData"
        :hide-base-style="customStyle !== CUSTOM_STYLE_DEFAULT"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActExplainDialog from 'press-ui/press-act-explain-dialog/press-act-explain-dialog.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-explain-dialog/demo-helper/style-list';


const PROPS_DATA = {
  title: '标题',
  content: Array.from({ length: 10 }).map((_, index) => `${index + 1}. Press UI 是一套支持 uni-app、普通 Vue 项目，同时兼容 Vue2 和 Vue3 的组件库`)
    .join('<br/>'),
};


export default {
  components: {
    PressActExplainDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActExplainDialog', STYLE_LIST, 'pvp-vertical'),
  ],
  data() {
    return {
      show: true,
      propsData: PROPS_DATA,
    };
  },

  mounted() {
  },
  methods: {
  },
};
</script>
<style scoped lang="scss">
</style>
