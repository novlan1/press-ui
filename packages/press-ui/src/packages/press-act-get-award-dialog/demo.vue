<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActGetAwardDialog
        :show.sync="show"
        :hide-tip-style="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :hide-base-style="customStyle !== CUSTOM_STYLE_DEFAULT"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActGetAwardDialog from 'press-ui/press-act-get-award-dialog/press-act-get-award-dialog.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-get-award-dialog/demo-helper/style-list';


export default {
  components: {
    PressActGetAwardDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActGetAwardDialog', STYLE_LIST),
  ],
  data() {
    return {
      show: true,
    };
  },

  mounted() {
  },
  methods: {
  },
};
</script>
<style scoped lang="scss">
</style>
