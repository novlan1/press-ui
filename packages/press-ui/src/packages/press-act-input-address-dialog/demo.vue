<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActInputAddressDialog
        :show.sync="show"
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :props-data.sync="propsData"
        @clickButton="clickButton"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActInputAddressDialog from 'press-ui/press-act-input-address-dialog/press-act-input-address-dialog.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-input-address-dialog/demo-helper/style-list';

const PROPS_DATA = {
  name: '名字',
  mobile: '123456789',
  zipcode: '888888',
  district: '地区1',
  detail_addr: '详细地址:',
  province: '',
  city: '',
};

export default {
  components: {
    PressActInputAddressDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActInputAddressDialog', STYLE_LIST, 'pvp-vertical'),
  ],
  data() {
    return {
      show: true,
      propsData: PROPS_DATA,
    };
  },

  mounted() {
  },
  methods: {
    clickButton() {
      console.log('[clickButton]', this.propsData);
      this.show = false;
    },
  },
};
</script>
<style scoped lang="scss">
</style>
