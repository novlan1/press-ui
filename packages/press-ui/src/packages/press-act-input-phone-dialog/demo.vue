<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActInputPhoneDialog
        :show.sync="show"
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :is-agree-protocol.sync="isAgreeProtocol"
        :need-phone-check="true"
        protocol-name="Press UI 协议"
        @clickButton="clickButton"
        @clickProtocol="clickProtocol"
        @clickUserProtocol="clickUserProtocol"
        @clickCloseButton="clickCloseButton"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActInputPhoneDialog from 'press-ui/press-act-input-phone-dialog/press-act-input-phone-dialog.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-input-phone-dialog/demo-helper/style-list';


export default {
  components: {
    PressActInputPhoneDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActInputPhoneDialog', STYLE_LIST),
  ],
  data() {
    return {
      show: true,
      isAgreeProtocol: false,
    };
  },

  mounted() {
  },
  methods: {
    clickButton() {
      this.onGTip('[clickButton]');
    },
    clickProtocol() {
      this.onGTip('[clickProtocol]');
    },
    clickUserProtocol() {
      this.onGTip('[clickUserProtocol]');
    },
    clickCloseButton() {
      this.onGTip('[clickCloseButton]');
    },
  },
};
</script>
<style scoped lang="scss">
</style>
