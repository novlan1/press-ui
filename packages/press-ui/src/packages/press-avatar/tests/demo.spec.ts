import Demo from '../demo.vue';
import { snapshotDemo } from '../../../../tests/unit/demo';


snapshotDemo(Demo, {
  // Demo 页面包含开启了随机背景色的头像组件，mock `Math.random` 函数以维持 snapshot 一致。
  beforeTest: () => {
    jest.spyOn(Math, 'random')
      .mockReturnValueOnce(0.67)
      .mockReturnValue(0.34);
  },
});
