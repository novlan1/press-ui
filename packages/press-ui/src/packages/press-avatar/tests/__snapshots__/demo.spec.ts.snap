// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div class="demo-wrap">
  <div>
    <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
      <!---->
      <!---->
    </div>
  </div>
  <div>
    <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent; margin-right: 20px;">
      <!---->
      <!---->
    </div>
    <div class="press-avatar press-avatar--square" style="width: 40px; height: 40px; background: transparent;">
      <!---->
      <!---->
    </div>
  </div>
  <div>
    <div class="press-avatar press-avatar--circle" style="width: 30px; height: 30px; background: transparent; margin-right: 20px;">
      <!---->
      <!---->
    </div>
    <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent; margin-right: 20px;">
      <!---->
      <!---->
    </div>
    <div class="press-avatar press-avatar--circle" style="width: 50px; height: 50px; background: transparent;">
      <!---->
      <!---->
    </div>
  </div>
  <div>
    <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: rgb(192, 196, 204); margin-right: 20px;">
      <!---->
      <div class=" press-icon-plus press-icon-plus-star-o" style="color: rgb(255, 255, 255); font-size: 18px;">
        <!---->
        <!---->
      </div>
    </div>
    <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: rgb(192, 196, 204);">
      <!---->
      <div class=" press-icon-plus press-icon-plus-like-o" style="color: rgb(255, 255, 255); font-size: 18px;">
        <!---->
        <!---->
      </div>
    </div>
  </div>
  <div>
    <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: rgb(192, 196, 204); margin-right: 20px;">
      <!----> <span style="color: rgb(255, 255, 255); text-align: center;">Y</span>
    </div>
    <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: rgb(134, 206, 250); margin-right: 20px;">
      <!----> <span style="color: rgb(255, 255, 255); text-align: center;">G</span>
    </div>
    <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: rgb(119, 177, 204);">
      <!----> <span style="color: rgb(255, 255, 255); text-align: center;">Êù®</span>
    </div>
  </div>
  <div>
    <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
      <!---->
      <!---->
    </div>
  </div>
  <div>
    <div class="press-avatar-group" style="margin-bottom: 16px;">
      <div class="press-avatar-group__item" style="margin-left: 0px; width: 40px; height: 40px;">
        <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -20px; width: 40px; height: 40px;">
        <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -20px; width: 40px; height: 40px;">
        <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -20px; width: 40px; height: 40px;">
        <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -20px; width: 40px; height: 40px;">
        <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
          <!---->
          <!---->
        </div>
        <div class="press-avatar-group__item__show-more"><span style="color: rgb(255, 255, 255); text-align: center;">
        2
      </span></div>
      </div>
    </div>
    <div class="press-avatar-group" style="margin-bottom: 16px;">
      <div class="press-avatar-group__item" style="margin-left: 0px; width: 40px; height: 40px;">
        <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -24px; width: 40px; height: 40px;">
        <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -24px; width: 40px; height: 40px;">
        <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -24px; width: 40px; height: 40px;">
        <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -24px; width: 40px; height: 40px;">
        <div class="press-avatar press-avatar--circle" style="width: 40px; height: 40px; background: transparent;">
          <!---->
          <!---->
        </div>
        <div class="press-avatar-group__item__show-more"><span style="color: rgb(255, 255, 255); text-align: center;">
        2
      </span></div>
      </div>
    </div>
    <div class="press-avatar-group" style="margin-bottom: 16px;">
      <div class="press-avatar-group__item" style="margin-left: 0px; width: 50px; height: 50px;">
        <div class="press-avatar press-avatar--circle" style="width: 50px; height: 50px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -25px; width: 50px; height: 50px;">
        <div class="press-avatar press-avatar--circle" style="width: 50px; height: 50px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -25px; width: 50px; height: 50px;">
        <div class="press-avatar press-avatar--circle" style="width: 50px; height: 50px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -25px; width: 50px; height: 50px;">
        <div class="press-avatar press-avatar--circle" style="width: 50px; height: 50px; background: transparent;">
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="press-avatar-group__item" style="margin-left: -25px; width: 50px; height: 50px;">
        <div class="press-avatar press-avatar--circle" style="width: 50px; height: 50px; background: transparent;">
          <!---->
          <!---->
        </div>
        <div class="press-avatar-group__item__show-more"><span style="color: rgb(255, 255, 255); text-align: center;">
        2
      </span></div>
      </div>
    </div>
  </div>
</div>
`;
